source/features/action-pr-link.tsx

```diff
@@ -13,7 +13,7 @@ function setSearchParameter(anchorElement: HTMLAnchorElement, name: string, valu
 async function addForRepositoryActions(prLink: HTMLAnchorElement): Promise<void> {
 	const prNumber = prLink.textContent.slice(1);
 
-	const runLink = prLink.closest('.Box-row')!.querySelector('a.Link--primary')!;
+	const runLink = prLink.closest('.Box-row')!.querySelector('a[href*="actions/runs"]')!;
```

Comment:
I still see the class, just not on the `a`

```suggestion
	const runLink = prLink.closest('.Box-row')!.querySelector('a:has(.Link--primary)')!;
```

<img width="446" alt="Screenshot" src="https://github.com/refined-github/refined-github/assets/1402241/8c5b887d-c91b-48ab-8eee-0f981816a45d">